src/components/Project/AragonClient.js

```diff
@@ -180,15 +181,17 @@ class AragonClient extends React.Component {
               <Entry>
                 <Module>
                   <div className="badge-container">
-                    <StyledBadge className="yellow">
-                      UX improvements
+                    <StyledBadge className="red">
```

Comment:
I think there is no class red to style the badge. May be we can use here the "blue" one cause we are not using it anymore (There is no more Flexible permissions badge)